mobile/lib/app/features/integrations/view/integration_page.view.dart

```diff
@@ -0,0 +1,203 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:triggo/app/features/integrations/integration.names.dart';
+import 'package:triggo/app/routes/routes_names.dart';
+import 'package:triggo/app/widgets/banner.triggo.dart';
+import 'package:triggo/app/widgets/button.triggo.dart';
+import 'package:triggo/mediator/integration.mediator.dart';
+import 'package:triggo/models/integration.model.dart';
+import 'package:triggo/models/integrations/discord.integration.model.dart';
+
+class IntegrationPage extends StatefulWidget {
+  const IntegrationPage({super.key});
+
+  final String title = 'Integration Screen';
+
+  @override
+  State<IntegrationPage> createState() => _IntegrationPageState();
+}
+
+class _IntegrationPageState extends State<IntegrationPage> {
+  @override
+  Widget build(BuildContext context) {
+    final IntegrationMediator integrationMediator =
+        RepositoryProvider.of<IntegrationMediator>(context);
+    final Future<List<Integration>> integrations =
+        integrationMediator.getIntegrations();
+
+    return Scaffold(
+      body: Padding(
+        padding: const EdgeInsets.all(8.0),
+        child: SafeArea(
+          child: Padding(
+            padding: const EdgeInsets.symmetric(horizontal: 8.0),
+            child: Column(
+              children: [
+                TriggoBanner(),
+                const SizedBox(height: 16.0),
+                Align(
+                  alignment: Alignment.centerLeft,
+                  child: Text(
+                    'Integrations',
+                    style: Theme.of(context).textTheme.titleLarge,
+                  ),
+                ),
+                const SizedBox(height: 4.0),
+                Expanded(
+                  child: Container(
+                    padding: const EdgeInsets.all(8.0),
+                    decoration: BoxDecoration(
+                      borderRadius: BorderRadius.circular(12.0),
+                      color: Colors.grey[200],
+                    ),
+                    child: Column(
+                      children: [
+                        Row(
+                          children: [
+                            Expanded(
+                              child: Padding(
+                                padding: const EdgeInsets.symmetric(
+                                    vertical: 8.0, horizontal: 4.0),
+                                child: IntegrationConnectionButton(),
+                              ),
+                            ),
+                          ],
+                        ),
+                        Expanded(
+                            child: IntegrationList(integrations: integrations)),
+                      ],
+                    ),
+                  ),
+                ),
+              ],
+            ),
+          ),
+        ),
+      ),
+    );
+  }
+}
+
+class IntegrationList extends StatelessWidget {
```

Comment:
Put an underscore before the classes you use only in this file
```suggestion
class _IntegrationList extends StatelessWidget {
```